docs/development.md

```diff
@@ -261,14 +261,22 @@ to build an image and a deployment config.
 
 ### Automated Deployment
 
-If you just want to have this automated, then jump to the [automated deployment
-steps in the userguide](userguide.md#automated-deployment) but be sure to
-specify your own image such as:
+If you just want to have this automated, then run the following command
+specifying your own image. This assumes you've used the steps [documented
+above](#setup-clusters-deploy-the-cluster-registry-and-federation-v2-control-plane) to
+set up two minikube clusters (`cluster1` and `cluster2`):
 
 ```bash
-./scripts/deploy-federation-latest.sh <containerregistry>/<username>/federation-v2:test cluster2
+./scripts/deploy-federation.sh <containerregistry>/<username>/federation-v2:test cluster2
 ```
 
+**NOTE:** You can list multiple joining cluster names in above command. Also,
```

Comment:
s/above command/the command above/

(No action required) This seems to be entirely duplicated in the userguide.  Is there an intent to reuse text snippets or use references in the future?

Reply:
This section, *Automated Deployment*, contains a slightly different command than what's captured in the user guide but the **NOTE** still applies. Because of that, I opted to repeat the text instead of adding a reference to go look at the **NOTE** in the other user guide's equivalent section in order to avoid any confusion.

It'd be nice to have a feature to maintain content in separate markdown files and use some sort of import/include statement to source and render imported markdowns whenever there is any re-use of text.